### ISSUE TITLE
preprocess data 

### ISSUE THREAD

**Issue by efferre99** [GoogleCloudPlatform/tensorflow-without-a-phd#29]
I have to preprocess images that have multi label.
In particular I have to project a neural network that not only classifies two different shape, triangle or circle, but also get their dimensions: the side, the radius etc.. It's seems with no sense, but for my application it has. Where can I find a similar example?

**Comment by martin-gorner** [comment]
The approach outlined in the tensorflow-planespotting example (object detection) does give you the size of the detected object. You will have to modify the YOLO head and the loss to get 2-way classification. Plaespotting only classifies airplanes and non-airplanes. I suggest you go back to the YOLO paper for that. It has the loss to be used for multi-label classification.
